product-cli/cmd/cluster/aws/create.go

```diff
@@ -0,0 +1,66 @@
+package aws
+
+import (
+	"context"
+	"github.com/spf13/cobra"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1beta1"
+	hypershiftaws "github.com/openshift/hypershift/cmd/cluster/aws"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "aws",
+		Short:        "Creates basic functional HostedCluster resources on AWS",
+		SilenceUsage: true,
+	}
+
+	opts.AWSPlatform = core.AWSPlatformOptions{
+		AWSCredentialsFile: "",
+		Region:             "us-east-1",
+		InstanceType:       "",
+		RootVolumeType:     "gp3",
```

Comment:
would these defaults silently apply? can we please mention their values in the cli commands description? Or is there any other way for the cli user to figure defaults?

Reply:
I'm not sure what you mean by silently apply. The CLI user can figure out the defaults by running the command with the help flag: `ex: hcp create cluster aws --help`. The 3rd value in the flag is the default value. When you run the help command, it shows you the default value after the description of the flag.